main/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,77 @@
+import AccordionItem, {AccordionData} from './AccordionItem';
+import {Animated, StyleProp, ViewStyle} from 'react-native';
+import React, {FC, ReactElement, useRef} from 'react';
+
+import styled from '@emotion/native';
+
+const Container = styled.View`
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const TOGGLE_ELEMENT_PROPERTY = 'toggleElement';
+export interface Styles {
+  titleContainer?: StyleProp<ViewStyle>;
+  bodyContainer?: StyleProp<ViewStyle>;
+}
+
+export interface AccordionBaseProps<T> {
+  data: T;
+  style?: ViewStyle;
```

Comment:
You'd better to use `StyleProps<ViewStyle>` since `ViewStyle` cannot accept style array like `style={[{}, {}]}`.

Reply:
Thanks~! I fixed it.